src/anomalib/models/image/fre/lightning_model.py

```diff
@@ -185,10 +185,9 @@ def trainer_arguments(self) -> dict[str, Any]:
         Returns:
             dict[str, Any]: Dictionary of trainer arguments:
                 - ``gradient_clip_val``: ``0``
-                - ``max_epochs``: ``220``
                 - ``num_sanity_val_steps``: ``0``
         """
-        return {"gradient_clip_val": 0, "max_epochs": 220, "num_sanity_val_steps": 0}
+        return {"gradient_clip_val": 0, "num_sanity_val_steps": 0}
```

Comment:
Is this change relevant to the issue?

Reply:
No but I saw that this parameter was overriding the max epochs value in the tests. And, according to our current design, this isn't mandatory (more like recommended). So, like other models recommended `max_epochs` is set in the model config.